src/Umbraco.Web.UI.Client/src/views/common/infiniteeditors/datatypesettings/datatypesettings.controller.js

```diff
@@ -10,7 +10,7 @@
 (function () {
     "use strict";
 
-    function DataTypeSettingsController($scope, dataTypeResource, dataTypeHelper, localizationService) {
+    function DataTypeSettingsController($scope, dataTypeResource, dataTypeHelper, localizationService, notificationsService, overlayService, formHelper) {
```

Comment:
```suggestion
    function DataTypeSettingsController($scope, dataTypeResource, dataTypeHelper, localizationService, notificationsService, formHelper) {
```

Looks like you do not need `overlayService`